cypress/e2e/Check.Full.Payment.Correct.js

```diff
@@ -0,0 +1,119 @@
+import dayjs from 'dayjs'
```

Comment:
Lets make sure the important pieces of this test are incorporated into `Normal_Paths` and remove this extra test.